src/toxcore/dht/client.rs

```diff
@@ -150,10 +150,10 @@ impl Client {
         res.push(
             ("255.255.255.255").parse().unwrap()
         );
-        // Ipv6 broadcast address
-        res.push(
-            ("FF02::1").parse().unwrap()
-        );
+        // TODO: Ipv6 broadcast address
```

Comment:
Why did you remove ipv6 addr?

Reply:
In dht_node.rs(example) there is no binding socket for IPv6.
So Ipv6 packet can't be sent using Ipv4 socket.

Reply:
As I see in LAN_Discovery.c#380 they check type of socket and then send packet to the right address.

Reply:
btw, they use IPv4-in-IPv6 to send to IPv4 broadcast address if socket is IPv6.